ansible/roles/containers/templates/atfncat.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-singularity exec {{ container_image }} /software/psrcat/psrcat "$@"
+exec singularity exec --bind /software:/software {{ container_image }} /opt/psrcat/psrcat "$@"
```

Comment:
I added this back in ... since I checked and you did use it everywhere. It's only the cases where you used singularity run that you didn't use the first exec. So I am guessing there was some issue this resolved.

Reply:
@kennedydane